engine/src/main/java/org/hibernate/search/bridge/builtin/time/impl/DurationBridge.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge.builtin.time.impl;
+
+import java.time.Duration;
+import java.util.Locale;
+
+import org.hibernate.search.bridge.TwoWayStringBridge;
+import org.hibernate.search.util.logging.impl.Log;
+import org.hibernate.search.util.logging.impl.LoggerFactory;
+
+/**
+ * Converts a {@link Duration} to a {@link String}.
+ * <p>
+ * The string is obtained concatenating the number of seconds with the nano of seconds.
+ * The values are padded with 0 to allow field sorting.
+ *
+ * @author Davide D'Alto
+ */
+public class DurationBridge implements TwoWayStringBridge {
```

Comment:
As durations are naturally a numeric value, I think it makes more sense to map it to a numeric field really.